examples/deep-peek/bower.json

```diff
@@ -1,8 +1,7 @@
-{
-  "name": "deep-peek",
-  "private": true,
-  "dependencies": {
-    "purescript-generics": "^0.7.0",
-    "purescript-halogen": "*"
-  }
-}
+{
+  "name": "deep-peek",
+  "private": true,
+  "dependencies": {
+    "purescript-halogen": "*"
+  }
+}

```

Comment:
Some of these diffs aren't very diff-y. Not sure why... line endings? Shouldn't be changing now for any reason I can think of.